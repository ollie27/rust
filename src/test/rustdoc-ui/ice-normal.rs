// compile-flags:-Z unstable-options -Z treat-err-as-bug
// failure-status: 101

fail
//~^ ERROR expected one of `!` or `::`, found `<eof>`
