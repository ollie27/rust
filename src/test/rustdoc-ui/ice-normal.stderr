error: expected one of `!` or `::`, found `<eof>`
  --> $DIR/ice-normal.rs:4:1
   |
LL | fail
   | ^^^^ expected one of `!` or `::` here

thread 'rustc' panicked at 'aborting due to `-Z treat-err-as-bug=1`', src/librustc_errors/lib.rs:546:13
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace.

error: internal compiler error: unexpected panic

note: the compiler unexpectedly panicked. this is a bug.

note: we would appreciate a bug report: https://github.com/rust-lang/rust/blob/master/CONTRIBUTING.md#bug-reports

note: rustc 1.38.0-dev running on x86_64-pc-windows-msvc

note: compiler flags: -Z threads=1 -Z ui-testing -Z unstable-options -Z unstable-options -Z treat-err-as-bug -C debuginfo=0

